components/esm/loadland.cpp

```diff
@@ -200,9 +201,9 @@ namespace ESM
             mLandData->mTextures[i] = 0;
         for (int i = 0; i < LAND_NUM_VERTS; ++i)
         {
-            mLandData->mColours[i*3+0] = -1;
-            mLandData->mColours[i*3+1] = -1;
-            mLandData->mColours[i*3+2] = -1;
```

Comment:
I wonder why compilers did not complain - mColours is an array of unsigned chars.